Description: Fix FTBFS against libsane-dev >= 1.0.25-3
 Fix missing sane-config
Author: Andreas Metzler <ametzler@bebt.de>
Reviewed-by: Jörg Frings-Fürst <debian@jff-webhosting.net>
Last-Update: 2017-01-28 
Index: trunk/configure.in
===================================================================
--- trunk.orig/configure.in
+++ trunk/configure.in
@@ -112,7 +112,10 @@ fi
 
 AC_CHECK_TOOL(RANLIB, ranlib, :)
 
-AM_PATH_SANE(1.0.0, HAVE_SANE=yes, )
+PKG_CHECK_MODULES([SANE], [sane-backends >= 1.0.0])
+PKG_CHECK_VAR([SANE_PREFIX], [sane-backends >= 1.0.0], [prefix])
+PKG_CHECK_VAR([SANE_LDFLAGS], [sane-backends >= 1.0.0], [ldflags],
+      [HAVE_SANE=yes])
 dnl Print error message if sane is missing
 if test "x${HAVE_SANE}" = "x"; then
   echo "**********************************************************************"
